### PR TITLE
Updated contributers hof

### LINKED ISSUE #8: Update Contributors HOF in Readme
Update contributors hof at the bottom of the readme.
This website can be used to generate repository hof

https://gh-hof.netlify.com/

### PR DESCRIPTION
**Pull request by darien-schettler**
Fixes #8 
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,7 @@ Read our contributing [guide](https://github.com/arshadkazmi42/currency-symbols/
 ## Contributors
 
 Thank you to all the contributors who help in making this project better :raised_hands:
-
-<a href="https://github.com/arshadkazmi42"><img src="https://github.com/arshadkazmi42.png" width="30" /></a>
+<a href="https://github.com/arshadkazmi42"><img src="https://github.com/arshadkazmi42.png" width="30" /></a><a href="https://github.com/DavidLeeR"><img src="https://github.com/DavidLeeR.png" width="30" /></a>
```
Comment (arshadkazmi42) [inline README.md]: Can  you add a line break here.

Comment (darien-schettler) [reply README.md]: Will do!

Comment (darien-schettler) [reply README.md]: Sorry for the double commit ! I just wanted to format the markdown a little prettier. I should have labelled the commit better.

**Comment by arshadkazmi42** [comment]
Congrats on landing your first PR

**Comment by darien-schettler** [comment]
> Congrats on landing your first PR

Thank you so much! I really appreciate your support!!

**Pull request merged**